pkg/storage/storagedb.go

```diff
@@ -52,7 +52,7 @@ func New(options *Options) (*StorageDB, error) {
 		}
 		dbpath := filepath.Join(options.DbPath, xid.New().String())
 
-		if err := os.MkdirAll(dbpath, 0644); err != nil {
+		if err := os.MkdirAll(dbpath, 0755); err != nil {
```

Comment:
Let's update this with `permissionutil.ConfigFolderPermission`. See: https://github.com/projectdiscovery/utils/blob/f110843046dd80c714ced0ba0857f0602e9f8c6f/permission/permission_file.go#L48

Reply:
Thanks for the suggestion, I've changed it.